test/constants.js

```diff
@@ -105,6 +105,56 @@ let testEntities = {
   }
 };
 
+let testCreateObservationsRequestBodies = {
+  CreateObservationsRequestOneObservation:
+  [
+    {
+      Datastream: { '@iot.id': 1 },
```

Comment:
I'm not sure. I believe it should exist. So I guess the '1' is a place holder for the actual iot.id of the Datastream that the test will create before testing the CreateObjerverations endpoint.